python/lsst/ts/watcher/model.py

```diff
@@ -230,6 +230,37 @@ def get_rules(self, name_regex):
         compiled_re = re.compile(name_regex)
         return [rule for name, rule in self.rules.items() if compiled_re.match(name) is not None]
 
+    def mute_alarm(self, name, duration, severity, user):
+        """Mute one or more alarms for a specified duration.
```

Comment:
"one ore more alarms for a specified duration at a specific severity"? Because you can mute multiple alarms with a regex, but you can do it for a single severity level.

Reply:
I'll say "for a specified duration and severity"